### DISCUSSION TITLE
Weekly updates 11-08-2021

### DISCUSSION THREAD

**Discussion by geromegrignon** [realworld-apps/realworld#799] [Weekly Updates]
# What did we do last week?

### V1

- FEAT: promote `api.realworld.io` API URL over the original `conduit.productionready.io` one in the documentation. #788 #790
- FIX: image property value in API profile endpoints #789
- FEAT: merged OpenAPI documentation and removed Swagger one
- FIX: GET profile always returning following to false #798

### V2

- FEAT: create V2 branch for RealWorld repository ([https://github.com/gothinkster/realworld/tree/v2](https://github.com/gothinkster/realworld/tree/v2))
- FEAT: new OpenAPI documentation hosted on [bump.sh](http://bump.sh) ([https://bump.sh/doc/realworld](https://bump.sh/doc/realworld))
- FEAT: add [bump.sh](http://bump.sh) GitHub Action to review and deploy the API documentation
- FEAT: create a repository for backend official implementation (https://github.com/gothinkster/node-express-prisma-official-app)

# What are we prioritizing this week?

- code review of https://github.com/gothinkster/node-express-prisma-official-app as v2 backend implementation to prepare it for contributions #790
- Create issue and pull request on each existing implementation using `conduit.productionredy.io` to change it to `api.realworld.io`

# Potential blockers?

- Unmaintained implementations for #790

**Comment by sultan3141** [comment]
6
